### PR TITLE
Bump sha-1 to v0.8.

### PR DESCRIPTION
**Pull request by dragostis**

### CODE REVIEW

**Comment by CAD97** [comment]
minimal-versions failure.

It [looks like block-buffer:0.7.0 isn't minimal-versions correct](https://travis-ci.org/pest-parser/pest/jobs/520197840#L576). The [type in question was made available in byteorder:1.1.0](https://docs.rs/byteorder/1.1.0/byteorder/type.BE.html).

Issues/PRs are being filed upstream: [block-buffer](https://github.com/RustCrypto/utils/pull/23).

Local workarounds (any one):
```toml
sha-1 = "0.7 - 0.8"
byteorder = "1.1"
```

File: meta/Cargo.toml [hunk 1]
```diff
@@ -23,4 +23,4 @@ maintenance = { status = "actively-developed" }
 travis-ci = { repository = "pest-parser/pest" }
 
 [build-dependencies]
-sha-1 = "0.7.0"
+sha-1 = "0.8"
```
Comment (CAD97) [inline meta/Cargo.toml]:
```suggestion
sha-1 = "0.7 - 0.8"
```

**Comment by CAD97** [comment]
Well, turns out Rust semver doesn't like that range syntax. ¯\\\_(ツ)_/¯

**Comment by dragostis** [comment]
@CAD97, I've opened a PR to update `byteorder` to `1.1` in block-buffer.

**Comment by CAD97** [comment]
Well, that makes [two of us, whoops](https://github.com/RustCrypto/utils/pull/23)

**Comment by dragostis** [comment]
Deleted mine.

**Comment by CAD97** [comment]
bors: r+

Let's land the version bump with the byteorder reference. We can then remove it again when sha-1 gets a new minimal-versions-compatible release.

**Pull request merged**